Panic when connection reset

```
panic: read tcp 127.0.0.1:4001->127.0.0.1:26913: read: connection reset by peer

goroutine 84 [running]:
main.handle_websocket.func3()
        /home/_nostr/gnost-relay/websocket.go:198 +0x113f
created by main.handle_websocket
        /home/_nostr/gnost-relay/websocket.go:134 +0xa0a
```

--- a/websocket.go
+++ b/websocket.go
@@ -195,7 +195,8 @@ func handle_websocket(cancel context.CancelFunc, wg *sync.WaitGroup, handshake *
 					n, err = conn.Read(mask_buf[:remaining])
 				}
 				if err != nil {
-					panic(err)
+					logger.Println(err)
+					return
 				}
 				remaining -= int64(n)
 				if header.Masked {

